### ISSUE TITLE
The website is down

### ISSUE THREAD

**Issue by emdash** [euglug/euglug_org#11]
euglug.org is down, and I haven't been able to contact anyone via meetup.com, for other reasons.

**Comment by laprice** [comment]
Yes. The person who took over the domain from me in 2019 did not renew it after their 2 year registration ran out. Someone registered it through godaddy but does not appear to be doing anything with it. The IRC channel is now libera.chat/#euglug

**Comment by emdash** [comment]
> The IRC channel is not libera.chat/#euglug

Did you mean "is now" or "is not"?
